students/bplanica/lesson04/mailroom2.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+
+# ------------------------------ #
+# Assignment 4 (Mailroom Part 2) for Python 210
+# Dev: Breeanna Planica
+# ChangeLog: (who, when, what)
+#   BPA, 8/11/2019, Created and tested script (Mailroom Part 1)
+#   BPA, 8/18/2019, Updated to add dictionaries and dispatch menu
+#                   Adding function to print emails to text files
+# ------------------------------ #
+
+# ----- DATA ----- #
+# ---------------- #
+donors = [{"Donor": "Bill Gates", "Donation": [326892.24, 326892.24]},
+{"Donor": "Mark Zuckerberg", "Donation": [5465.39, 5465.39, 5465.39]},
+{"Donor": "Jeff Bezo", "Donation": [877.33]},
+{"Donor": "Paul Allen", "Donation": [236.14, 236.14, 236.14]},
+{"Donor": "M Bezo", "Donation": [110463.25]}] # holds original data set to start with
```

Comment:
This works, but it's a list containing dictionaries.  Ideally, you'd want a single dictionary with the donors as keys.  For example:

`def get_donor_db():
    return {'bill gates': ("Bill Gates", [326892.24, 326992.24]),
            'mark zuckerberg': ("Mark Zuckerberg", [5465.39, 5465.39, 5465.39]),
            'etc, etc,
            }`